src/api/app/components/workflow_run_request_action_filter_component.html.haml

```diff
@@ -0,0 +1,12 @@
+= form_with(url: token_workflow_runs_path(@token_id), method: :get, local: true, class: 'form-inline') do
+  = label_tag('request_action', 'Action', class: 'sr-only')
+  .input-group.mr-sm-2
+    .input-group-prepend
+      .input-group-text
+        Action
+    = select_tag('request_action',
```

Comment:
@rubhanazeem its a small thing, but the select box doesn't have the round edges on the right side 
![image](https://user-images.githubusercontent.com/22001671/162952948-9909fe09-fde6-4131-96d4-01711cfdd6e8.png)
at other places we have it 
![image](https://user-images.githubusercontent.com/22001671/162953117-d72773bf-f588-4973-a7c4-545c10f5fceb.png)

Reply:
would be cool to submit the form on select, since you can only choose one filter there is no need to first select and then submit through the `Apply` button. But that could be a follow up as well

Reply:
> would be cool to submit the form on select, since you can only choose one filter there is no need to first select and then submit through the `Apply` button. But that could be a follow up as well

I know we can have the form submitted on change but that's not going to be a clean solution because we also have pagination on the same page. Pagination doesn't update automatically with js request. That's why I moved to this solution.